conf/webpack.config.js

```diff
@@ -138,6 +139,12 @@ function getConfig(isReactExternalized) {
                     flags: 'g',
                 },
             },
+            {
+                test: /\.(js|mjs)$/,
+                loader: 'babel-loader',
+                include: /(node_modules)/,
```

Comment:
Seems like this could live directly in `test`.

Reply:
It's conventional to specify the file extensions in `test` and the paths in `include`/`exclude` (through either a regex or `path.resolve()`), but it's not enforced: https://webpack.js.org/configuration/module/#condition

Reply:
I don't usually see `include` and `exclude` get used simultaneously, since it can be a little confusing. If it works, though, I guess it's fine.